alternator/conditions.cc

```diff
@@ -179,6 +181,59 @@ static bool check_BEGINS_WITH(const rjson::value* v1, const rjson::value& v2) {
     return val1.substr(0, val2.size()) == val2;
 }
 
+static std::string_view get_strv(const rjson::value& v) {
```

Comment:
nitpick: I'm not a big fan of this name, it looks like it's taken straight from the 8086 instruction set :-). `to_string_view()`?

Reply:
Done.